### PR TITLE
DOCS: Moving skills and knowledge guides to taxonomy

### PR DESCRIPTION
**Pull request by kelbrown20**
**Description:** Moving the skills_guide and knowledge_guide files to the taxonomy repo. Removing the files in community and fixing the links. 

**Additional info:** 
Updates in taxonomy in this PR - https://github.com/instructlab/taxonomy/pull/777

### CODE REVIEW

**Comment by jjasghar** [comment]
YAY! 👍

**Review by jjasghar — APPROVED** [review]

**Comment by bjhargrave** [comment]
We probably should not merge this until after https://github.com/instructlab/taxonomy/pull/777 is merged.

File: docs/README.md
Comment (bjhargrave) [inline docs/README.md:17]:
```suggestion
- [Knowledge guide](https://github.com/instructlab/taxonomy/blob/main/docs/KNOWLEDGE_GUIDE.md)
```

File: docs/README.md
Comment (bjhargrave) [inline docs/README.md:13]:
```suggestion
- [Skills guide](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md)
```

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**
